pkg/bwagreement/server.go

```diff
@@ -42,6 +42,17 @@ type UplinkStat struct {
 	TotalTransactions int
 }
 
+//SavedOrder allows use of dbx.Bwagreement without the namespace
+type SavedOrder struct {
+	Serialnum     string
```

Comment:
`SerialNumber []byte`

Reply:
this is a string everywhere else in the codebase